blueocean-dashboard/src/main/js/components/RunDetailsChanges.jsx

```diff
@@ -68,7 +58,7 @@ export default class RunDetailsChanges extends Component {
             <Table headers={headers} className="changeset-table">
                 { changeSet.map(commit => (
                     <tr key={commit.commitId}>
-                        <td><CommitLink {...commit} /></td>
+                        <td><CommitHash commitId={commit.commitId} url={commit.url} /></td>
```

Comment:
is there a jsspec for RunDetailsChanges? Does it cover CommitHashes? Might be worth creating a test if its not a huge hassle.